lib/rspec/shell/expectations/call_log.rb

```diff
@@ -38,7 +37,7 @@ def get_argument_count(*expected_argument_series, sub_command_series: [], positi
         end
 
         def called_with_no_args?
-          load_call_log_list.first["args"].nil?
+          load_call_log_list.first["args"].to_a.empty?
```

Comment:
I'm not 100% sure on this change, but I think we may have a bug the way it was before